api/integrations/github/views.py

```diff
@@ -25,6 +28,22 @@
 from organisations.permissions.permissions import GithubIsAdminOrganisation
 
 
+def github_webhook_payload_is_valid(payload_body, secret_token, signature_header):
```

Comment:
Type hints

Reply:
Also, this function should not be part of views

Reply:
I put it there because It's a helper used to validate a request header so it looked very view-specific for me.  Where would you suggest to move it?

Reply:
signature.py

Reply:
Done. Moved to `helpers.py`